include/openenclave/enclave.h

```diff
@@ -724,6 +724,61 @@ oe_enclave_t* oe_get_enclave(void);
  */
 oe_result_t oe_random(void* data, size_t size);
 
+/**
+ * oe_gen_tls_cert.
+ *
+ * This function generates a self-signed certificate with an embedded quote from
+ * the underlying enclave.
+ *
+ * @param issuer_key: a private key used to sign this certificate
```

Comment:
Can we please use the direction arguments [in] or [out] or [in,out] to param to better document the syntax in any public headers?  See http://www.doxygen.nl/manual/commands.html#cmdparam

Reply:
addressed for those new APIs in this PR.
https://github.com/microsoft/openenclave/issues/1804 is opened to track the work for updating all other existing OE SDK API